app/assets/javascripts/components/campaign/campaign_ores_plot.jsx

```diff
@@ -37,7 +38,7 @@ const CampaignOresPlot = createReactClass({
 
   fetchFilePath() {
     $.ajax({
-      url: `/campaigns/${this.props.match.params.campaign_slug}/ores_data.json`,
+      url: '/campaigns/:campaign_slug/ores_data.json',
```

Comment:
This format only works as part of the `path` for a `Route` component; it uses pattern matching to find what the campaign slug is in the current path, and passes that as part of the `match` prop to the component it renders, if the route gets rendered via `component=` rather than `render=`.

Here, you can get the slug from the campaign prop instead, since that's how you've set it up so far. `/campaigns/${this.props.campaign.slug}/ores_data.json`